### ISSUE TITLE
Let /premium be only executed when the executor is a valid premium player.

### ISSUE THREAD

**Issue by Phreag** [games647/FastLogin#159]

**Comment by games647** [comment]
Thers is noway to verify that if you are already online. We can only check if the name is used by a premium account or request a relogin to proof that you own this account.

**Comment by Eufranio** [comment]
Cache it before the player actually log in (before FastLogin handle the state), and look on the cached data later?

**Comment by games647** [comment]
>Cache it before the player actually log in (before FastLogin handle the state),

Cache the offline mode login? Please explain

**Comment by Phreag** [comment]
Maybe do it like this:
Player Types /premium and gets kicked with the message "please log in again to activate Premium Mode" and if the player does not relog within a given time using a premium account it will be reset back to cracked.

**Comment by msmanui** [comment]
> Maybe do it like this:
> Player Types /premium and gets kicked with the message "please log in again to activate Premium Mode" and if the player does not relog within a given time using a premium account it will be reset back to cracked.

Yes please! This would be extremely useful and save me tons of tickets of people who make /premium just for curiosity and they are not even premium.

**Comment by JunederZ** [comment]
Bump +1
